frontend_app_kanban/src/component/KanbanCard.jsx

```diff
@@ -39,12 +39,41 @@ function KanbanCard (props) {
     )
   }
 
+  const luminance = (color) => {
+    const rgbHex = color.replace('#', '')
+    const red = parseInt(rgbHex.substr(0, 2), 16)
+    const green = parseInt(rgbHex.substr(2, 2), 16)
+    const blue = parseInt(rgbHex.substr(4, 2), 16)
+    const rgbDec = [red, green, blue]
+    var rgb = rgbDec.map(c => {
+      c /= 255
+      return c < 0.03928 ? c / 12.92 : Math.pow((c + 0.055) / 1.055, 2.4)
+    })
+    return (0.2126 * rgb[0] + 0.7152 * rgb[1] + 0.0722 * rgb[2])
+  }
+
+  // INFO - FS - 2024-03-25 - Use the contrast ratio to determine witch text color use : https://github.com/tracim/tracim/issues/6356
+  // return true if white text color is better than black for this background
+  const textColorNeededFromBackground = (backColor) => {
```

Comment:
Thinking about it now, but shouldn't this function be put in frontend_lib helpers ?

It will be needed in logbooks, and maybe other future contents. Could be good to have it accessible everywhere since it's not tied directly to kanban (only takes a color as a parameter)

Reply:
done